docs/source/gen/flytectl_get_execution.rst

```diff
@@ -64,7 +60,7 @@ Options
       --filter.asc                     Specifies the sorting order. By default flytectl sort result in descending order
       --filter.field-selector string   Specifies the Field selector
       --filter.limit int32             Specifies the limit (default 100)
-      --filter.sort-by string          Specifies which field to sort results  (default "created_at")
+      --filter.sort-by string          Specifies which field to sort result by 
```

Comment:
@evalsocket can we check how the default info got removed. Same here and in other places

Reply:
I removed it from the default value. The change is correct